pkg/client/e2e.go

```diff
@@ -27,16 +27,15 @@ import (
 
 	"github.com/heptio/sonobuoy/pkg/client/results"
 	"github.com/heptio/sonobuoy/pkg/client/results/e2e"
-	"github.com/onsi/ginkgo/reporters"
```

Comment:
Now use our own type which is an amalgamation of the ginkgo/reporters and the jstemmer types. Paired with a custom unmarshaller, this allows us to handle both cases with a single type.